plugin-server/src/worker/ingestion/event-pipeline/extractHeatmapDataStep.ts

```diff
@@ -172,8 +185,21 @@ function extractScrollDepthHeatmapData(event: PreIngestionEvent): RawClickhouseH
                     )
                     .filter((x): x is RawClickhouseHeatmapEvent => x !== null)
             )
+        } else {
+            await captureIngestionWarning(
+                runner.hub.kafkaProducer,
+                teamId,
+                'rejecting_heatmap_data_with_invalid_items',
```

Comment:
purposefully not capturing the data here because it might be large, this at least gives us the hook on where to investigate